nc-news/src/App.jsx

```diff
@@ -1,5 +1,45 @@
-import { useState } from "react";
+import { useState, useEffect } from "react";
+import { Routes, Route } from "react-router-dom";
+import Header from "./components/Header";
+import NavBar from "./components/NavBar";
+import SideBar from "./components/SideBar";
+import AllArticles from "./components/AllArticles";
+import Profile from "./components/Profile";
+import About from "./components/About";
+import { fetchAllArticles } from "../utils/utils";
 
-function App() {}
+function App() {
+  const [tripleBarStatus, setTripleBarStatus] = useState("header--inactive");
+  const [sideBarStatus, setSideBarStatus] = useState("sidebar--inactive");
+  const [allArticles, setAllArticles] = useState([]);
+
+  useEffect(() => {
+    fetchAllArticles().then((res) => {
+      setAllArticles(res);
+    });
+  }, []);
+
```

Comment:
i don't think you need allArticles to be fetched this high up in the app... i could be wrong but this useEffect and the state declaration can go in AllArticles component